### PR TITLE
LPS-191869 | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-191869](https://liferay.atlassian.net/browse/LPS-191869)**

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
 <tr>
 	<td>PREVIEW_TITLE</td>
-	<td>//span[@class='h3 task-content-title']</td>
+	<td>//*[contains(@class, 'task-content-title')]</td>
```
Comment (davisaints) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path:46]: The old path was working for the preview title in both "My workflow tasks" and "My submissions." But due to [this](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/edit_workflow_task.jsp#L255) change, they stopped working. I just made it usable again in both cases

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path [hunk 2]
```diff
@@ -43,12 +43,12 @@
...
 	<td></td>
 </tr>
 <tr>
 	<td>PREVIEW_CONTENT</td>
-	<td>//span[@class='h3 task-content-title']/../div[2] | //span[@class='h3 task-content-title']/../../div</td>
+	<td>//div[contains(@class, 'asset-summary')]</td>
```
Comment (davisaints) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path:51]: We can just use this

**Comment by patriciajeanperez** [comment]
ci:test:workflow

**Comment by davisaints** [comment]
ci:test:workflow

**Comment by davisaints** [comment]
ci:test:workflow

**Comment by davisaints** [comment]
No related failures, should be good to go 

cc/ @patriciajeanperez

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path [hunk 3]
```diff
@@ -43,12 +43,12 @@
...
 <tr>
 	<td>PREVIEW_TITLE</td>
-	<td>//span[@class='h3 task-content-title']</td>
+	<td>//*[contains(@class, 'task-content-title')]</td>
 	<td></td>
 </tr>
 <tr>
 	<td>PREVIEW_CONTENT</td>
-	<td>//span[@class='h3 task-content-title']/../div[2] | //span[@class='h3 task-content-title']/../../div</td>
+	<td>//*[contains(@class, 'web-content') or contains(@class, 'dynamic') or contains(@class, 'blogs') or contains(@class, 'kaleo-process') or contains(@class, 'comments')]/../../../following-sibling::div | //*[contains(@class, 'documents-and-media')]/../following-sibling::span[contains(@class, 'taglib-text')]</td>
```
Comment (davisaints) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path:51]: The preview content is different for each component's content, here I am making it work for all our tests

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**